src/helpers/helpers.js

```diff
@@ -33,24 +33,27 @@ function _sortAndNameLicenses (licenses) {
  * Gets the user details from the cookie
  * @returns {{}}
  */
-export function getCookieUserDetails () {
+export function getCookieUserDetails (katConfig, licenceId) {
   // if the data has not been retrieved
-  if (licenceData === undefined) {
+  // if (licenceData === undefined) { - not sure why we needed it here
```

Comment:
😬 🤷‍♂️

Reply:
Let's keep it commented for now then, just in case